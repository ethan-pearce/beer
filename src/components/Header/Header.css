.header {
    background-color: black;
    color: white;
    padding: 16px;
    position: sticky;
    top: 0;
    z-index: 1;
}