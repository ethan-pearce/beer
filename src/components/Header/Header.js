import './Header.css';

const Header = () => {
  return (
    <header className={'header'}>
      <h1>Speciality Beers</h1>
    </header>
  );
}

export default Header;