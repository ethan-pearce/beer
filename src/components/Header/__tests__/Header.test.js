import React from 'react'
import { render } from '@testing-library/react';
import Header from '../Header'
import { BrowserRouter as Router } from "react-router-dom";

describe('Header Component', () => {
    it('should render', () => {
        const { container } = render(<Router><Header /></Router>)

        expect(container.getElementsByClassName('header').length).toBe(1)
        expect(container.getElementsByClassName('headerTitle').length).toBe(1)
        expect(container).toMatchSnapshot()
    })
})