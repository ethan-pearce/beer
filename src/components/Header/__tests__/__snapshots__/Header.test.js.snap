// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      Speciality Beers
    </h1>
  </header>
</div>
`;
